frontend/mono-ui/web/rainmaker/dev-packages/egov-firenoc-dev/src/ui-config/screens/specs/fire-noc/applyResource/nocDetails.js

```
@@ -17,6 +17,10 @@ import {
 } from "../../../../../ui-utils/commons";
 import "./index.css";
 import { onchangeOfTenant } from "./propertyLocationDetails";
+export const getTenantId = () => {
+  let tenant=sessionStorage.getItem('Digit.Employee.tenantId');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'tenant' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)